test/functional/custom-decorators.spec.ts

```diff
@@ -1,261 +1,241 @@
-import "es6-shim";
 import {Validator} from "../../src/validation/Validator";
 import {ValidationArguments} from "../../src/validation/ValidationArguments";
 import {registerDecorator} from "../../src/register-decorator";
 import {ValidationOptions} from "../../src/decorator/ValidationOptions";
 import {ValidatorConstraint} from "../../src/decorator/decorators";
 import {ValidatorConstraintInterface} from "../../src/validation/ValidatorConstraintInterface";
 
-import {should, use } from "chai";
-
-import * as chaiAsPromised from "chai-as-promised";
-
-should();
-use(chaiAsPromised);
-
-// -------------------------------------------------------------------------
-// Setup
-// -------------------------------------------------------------------------
-
 const validator = new Validator();
 
-// -------------------------------------------------------------------------
-// Specifications: common decorators
-// -------------------------------------------------------------------------
```

Comment:
Please remove or keep these comments in all spec files :)

Reply:
I will leave this one for you to fix ;)